src/natcap/invest/utils.py

```diff
@@ -715,41 +710,36 @@ def read_csv_to_dataframe(
     column names that don't exactly match the specification. Strips
     leading/trailing whitespace from data entries as well.
 
+    Also sets custom defaults for some kwargs passed to ``pandas.read_csv``.
+
     Args:
-        path (string): path to a CSV file
+        path (str): path to a CSV file
         to_lower (bool): if True, convert all column names to lowercase
-        sep: separator to pass to pandas.read_csv. Defaults to None, which
-            lets the Python engine infer the separator (if engine='python').
-        encoding (string): name of encoding codec to pass to `pandas.read_csv`.
-            Defaults to None. Setting engine='python' when encoding=None allows
-            a lot of non-UTF8 encodings to be read without raising an error.
-            Any special characters in other encodings may get replaced with the
-            replacement character.
-            If encoding=None, and the file begins with a BOM, the encoding gets
-            set to 'utf-8-sig'; otherwise the BOM causes an error.
-        engine (string): kwarg for pandas.read_csv: 'c', 'python', or None.
-            Defaults to 'python' (see note about encoding).
         expand_path_cols (list[string])): if provided, a list of the names of
             columns that contain paths to expand. Any relative paths in these
             columns will be expanded to absolute paths. It is assumed that
             relative paths are relative to the CSV's path.
-
-        **kwargs: any kwargs that are valid for ``pandas.read_csv``
+        sep: kwarg of ``pandas.read_csv``. Defaults to None, which
+            lets the Python engine infer the separator
+        engine (str): kwarg of ``pandas.read_csv``. The 'python' engine
+            supports the sep=None option.
+        encoding (str): kwarg of ``pandas.read_csv``. Using the 'utf-8-sig'
+            encoding handles UTF-8 with or without BOM.
+        encoding_errors (str): kwarg of ``pandas.read_csv``. The 'replace'
+            option replaces unknown characters with a placeholder.
+        **kwargs: additional kwargs will be passed to ``pandas.read_csv``
 
     Returns:
         pandas.DataFrame with the contents of the given CSV
 
     """
-    # Check if the file encoding is UTF-8 BOM first
-    # allow encoding kwarg to override this if it's provided
-    if not encoding and has_utf8_bom(path):
-        encoding = 'utf-8-sig'
     try:
         dataframe = pandas.read_csv(
-            path, engine=engine, encoding=encoding, sep=sep, **kwargs)
+            path, sep=sep, engine=engine, encoding=encoding,
+            encoding_errors=encoding_errors, **kwargs)
     except UnicodeDecodeError as error:
         LOGGER.error(
-            f'{path} must be encoded as utf-8 or ASCII')
+            f'{path} must be encoded as UTF-8 or ASCII')
```

Comment:
To test my understanding...is it possible to get here with the default of `encoding_errors='replace'`? Or is this only catching errors if we were to use `encoding_errors='strict'`?

Reply:
I think it would only happen with `encoding_errors='strict'`.